integration-cli/daemon/daemon.go

```diff
@@ -678,6 +679,23 @@ func (d *Daemon) FindContainerIP(id string) (string, error) {
 	return strings.Trim(out, " \r\n'"), nil
 }
 
+// InspectContainer inspects a container
+func (d *Daemon) InspectContainer(ctx context.Context, id string) (*container.Config, *container.HostConfig, error) {
```

Comment:
These tests now probably have to be moved to the `/integration` suite, not `/integration-cli/`; https://github.com/moby/moby/blob/master/integration/service/create_test.go